godot-macros/src/derive_godot_class.rs

```diff
@@ -174,11 +174,46 @@ impl Field {
 
 struct ExportedField {
     field: Field,
-    getter: String,
-    setter: String,
+    getter: GetterSetter,
+    setter: GetterSetter,
     hint: Option<ExportHint>,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+enum GetterSetter {
+    /// Getter/setter should be omitted, field is write/read only.
+    Omitted,
+    /// Trivial getter/setter should be autogenerated.
+    Generated,
+    /// Getter/setter is hand-written by the user, and here is its name.
+    Custom(String),
+}
+
+impl GetterSetter {
+    fn parse(parser: &mut KvParser, key: &str) -> ParseResult<Self> {
+        Ok(match parser.handle_any(key) {
+            // No `get` argument
+            None => GetterSetter::Omitted,
+            // `get` without value
+            Some(KvValue::None) => GetterSetter::Generated,
+            // `get = literal`
+            Some(KvValue::Lit(name_lit)) => {
+                let name = string_lit_contents(&name_lit).ok_or_else(|| {
+                    venial::Error::new(format!(
+                        "argument to {key} must be a string literal, got: {name_lit}"
+                    ))
```

Comment:
Could you reuse `bail` here? In that case, you also pass a _span_, so that the resulting compiler error can point directly to some tokens in the source code.

Reply:
Done.